packages/babel-plugin/src/index.tsx

```diff
@@ -72,10 +122,7 @@ export default declare<State>((api) => {
           }
 
           (['styled', 'ClassNames'] as const).forEach((apiName) => {
-            if (
-              state.compiledImports &&
-              specifier.node?.imported.name === apiName
-            ) {
+            if (state.compiledImports && specifier.node?.imported.name === apiName) {
```

Comment:
hmm 🤔. Wondering why I got this diff. Is prettier not working under pre-commit hook? 🤔

Reply:
weird!